src/StaticFAQ/ArticleFeedback/ScreenThankyou.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+
+import * as React from 'react';
+import css from 'styled-jsx/css';
+import { Close } from '@kiwicom/orbit-components/lib/icons';
+
+import { Box, ContactPageLink } from '../../common';
+
+const style = css`
+  div.thank-you {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    height: 251px;
+  }
+  div.close-icon {
+    position: absolute;
+    top: 8px;
+    right: 8px;
+  }
+  div.message-1,
+  div.message-2 {
```

Comment:
I think this should be **bold** too

Reply:
Disagree again :smile: 
![image](https://user-images.githubusercontent.com/2466887/39757396-00231504-52cd-11e8-92f8-bdb9c3e9e416.png)